{"name": "test_6_get_unknown_by_id_not_found", "status": "passed", "start": 1710525159030, "stop": 1710525160190, "uuid": "8cc45920-0f92-4bd7-b4af-7099327130f6", "historyId": "f6ea42f65744e587c0e6843e8ce3d238", "testCaseId": "f6ea42f65744e587c0e6843e8ce3d238", "fullName": "Tests.test_api.TestLogin#test_6_get_unknown_by_id_not_found", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('api_fixture')"}, {"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-UEVTOM8"}, {"name": "thread", "value": "5224-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_api"}]}